src/SDKs/Search/DataPlane/Microsoft.Azure.Search/Microsoft.Azure.Search.csproj

```diff
@@ -7,7 +7,7 @@
     <AssemblyTitle>Microsoft Azure Search Library</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Search</AssemblyName>
     <Version>9.0.0</Version>
-    <PackageReleaseNotes>This is the newest major version of the Azure Search .NET SDK, based on version 2019-05-06 of the Azure Search REST API. New in this GA version is support for Cognitive Search, complex types for richer data modeling, the Autocomplete API, and additional parsing modes for Blob indexers. See this article for help on migrating to the latest version: http://aka.ms/search-sdk-upgrade.</PackageReleaseNotes>
+    <PackageReleaseNotes>Latest release fixes deadlocking issue and exposes optional Skill.Name field</PackageReleaseNotes>
```

Comment:
Please don't delete the other notes -- those are still applicable. Just add this to the end:

> This release includes a fix for the deadlock issue (`https://github.com/Azure/azure-sdk-for-net/issues/6254`) and exposes the new optional Skill.Name property.

Reply:
Actually, you can drop the part about the deadlock fix. I'll make that change in my own PR, so we can decouple them.